src/bridge/mod.rs

```diff
@@ -153,6 +165,44 @@ async fn start_process(mut receiver: UnboundedReceiver<UiCommand>) {
         .await
         .ok();
     }
+
+    nvim.set_client_info(
+        "neovide",
+        vec![(Value::from("major"), Value::from(0)), (Value::from("minor"), Value::from(6))],
+        "ui",
+        vec![],
+        vec![]
+    ).await.ok();
+
+    // parsing the client list to get the channel id for neovide
+    let neovide_channel: u64 =
+        nvim.list_chans().await.ok()
+        .and_then(|chans| {
+            let (_, id) = chans.iter().find_map(|chan| {
```

Comment:
This may be heretical, but I wonder if this could be simplified by just using a for loop. To my eyes these iter, find_map, as_map, find calls look pretty confusing.